tests/cms/test_index.py

```diff
@@ -0,0 +1,10 @@
+import pytest
```

Comment:
Please ignore the test / this file. I'm still trying to wrap my head around the required tests